README.md

```diff
@@ -120,6 +120,7 @@ You can configure the `connectTimeout` and `clientTimeout` properties on the `ne
 The plugin does the following:
 
 - configure a Maven artifact repository for each repository defined in the `nexusPublishing { repositories { ... } }` block in each subproject that applies the `maven-publish` plugin
+- creates a `get${repository.name.capitalize()}StagingProfile` task that either retrieves the staging profile id from the remote Nexus repository or from the configuration property `stagingProfileId` in  `nexusPublishing { repositories { myRepository { ... } } }`. Specifying the configuration property rather than relying on the API call is considered a performance optimization.  
```

Comment:
I think we should name the task in a way that makes it clearer that it makes a call to a server, e.g. `retrieve${repository.name.capitalize()}StagingProfileId` or `find${repository.name.capitalize()}StagingProfileId`.